docs/resources/rdb_instance.md

```diff
@@ -155,75 +154,75 @@ and if you are using `bssd` storage, you should increase the volume size before
 
 - `backup_schedule_frequency` - (Optional) Backup schedule frequency in hours.
 
 - `backup_schedule_retention` - (Optional) Backup schedule retention in days.
 
-- `backup_same_region` - (Optional) Boolean to store logical backups in the same region as the database instance.
+- `backup_same_region` - (Optional) Boolean to store logical backups in the same region as the Database Instance.
 
 ### Settings
 
 - `settings` - (Optional) Map of engine settings to be set. Using this option will override default config.
 
 - `init_settings` - (Optional) Map of engine settings to be set at database initialisation.
 
-~> **Important:** Updates to `init_settings` will recreate the Database Instance.
+~> **Important** Updates to `init_settings` will recreate the Database Instance.
 
-Please consult the [GoDoc](https://pkg.go.dev/github.com/scaleway/scaleway-sdk-go@v1.0.0-beta.9/api/rdb/v1#EngineVersion) to list all available `settings` and `init_settings` for the `node_type` of your convenience.
+-> **Note** Refer to the [GoDoc](https://pkg.go.dev/github.com/scaleway/scaleway-sdk-go@v1.0.0-beta.9/api/rdb/v1#EngineVersion) to list all available `settings` and `init_settings` for the `node_type` of your convenience.
 
 ### Endpoints
 
-- `private_network` - List of private networks endpoints of the database instance.
+- `private_network` - List of Private Networks endpoints of the Database Instance.
+
+  - `pn_id` - (Required) The ID of the Private Network.
+  - `ip_net` - (Optional) The IP network address within the private subnet. This must be an IPv4 address with a CIDR notation. If not set, The IP network address within the private subnet is determined by the IP Address Management (IPAM) service.
+  - `enable_ipam` - (Optional) If true, the IP network address within the private subnet is determined by the IP Address Management (IPAM) service.
 
-    - `pn_id` - (Required) The ID of the private network.
-    - `ip_net` - (Optional) The IP network address within the private subnet. This must be an IPv4 address with a CIDR notation. If not set, The IP network address within the private subnet is determined by the IP Address Management (IPAM) service.
-    - `enable_ipam` - (Optional) If true, the IP network address within the private subnet is determined by the IP Address Management (IPAM) service.
+~> **Important** One of `ip_net` or `enable_ipam=true` must be set.
 
-~> **Important:** One of `ip_net` or `enable_ipam=true` must be set.
+~> **Important** Updates to `private_network` will recreate the Instance's endpoint
 
-~> **Important:** Updates to `private_network` will recreate the Instance's endpoint
-  
-~> **NOTE:** Please calculate your host IP using [cidrhost](https://developer.hashicorp.com/terraform/language/functions/cidrhost). Otherwise, let IPAM service
+~> **Note** You can calculate your host IP using [cidrhost](https://developer.hashicorp.com/terraform/language/functions/cidrhost). Otherwise, let IPAM service
```

Comment:
```suggestion
-> **Note** You can calculate your host IP using [cidrhost](https://developer.hashicorp.com/terraform/language/functions/cidrhost). Otherwise, let IPAM service
```
Let's use blue color as the other notes ([format](https://developer.hashicorp.com/terraform/registry/providers/docs#callouts))